app/assets/javascripts/notebook_actions.js.erb

```diff
@@ -218,6 +218,63 @@ $(document).ready(function(){
     return false;
   });
 
+  /* ===== Delete External Resource ===== */
+  $('.external-resources .delete-icon').on('click',function(e){
+    e.preventDefault();
+    $('#confirmationModal').modal('show');
+    $('#confirmationModalForm').addClass('delete-resource');
+    var text = 'Delete the resource: "' + $(this).attr('data-resource-title') +  '" from notebook: "' + $(this).attr('data-notebook') + '".'
+    $('#confirmation1').text(text);
+    $('#confirmationModalForm #inAdditionText').addClass('hide').siblings('.additional-confirmation').text('').addClass('hide');
```

Comment:
Click to delete external resource
Click Cancel
Click to delete Notebook
Additional Info list is missing in dialog